components/centraldashboard/public/components/main-page.pug

```diff
@@ -1,5 +1,5 @@
-iron-ajax(auto, url='/api/platform-info', handle-as='json',
-    on-response='_onPlatformInfoResponse')
+iron-ajax(auto, url='/api/env-info', handle-as='json',
```

Comment:
You'll need to update the unit test that mocks this API response.

Reply:
Done!